### DISCUSSION TITLE
Color field interface must support alpha channel (transparency)

### DISCUSSION THREAD

**Discussion by adanielyan** [directus/directus#8990] [Feature Requests]
Currently when Color is selected as a string field interface, it displays a color picker without color transparency control in both hex and RGB formats. The text input doesn't even allow adding the alpha value manually (e.g. #ff0000**55**). It is important to allow for colors with transparency as they are widely used when building websites.

**Comment by benhaynes** [comment]
Good feature request, thank you.

**Comment by adanielyan** [comment]
Just created a pull request: https://github.com/directus/directus/pull/9224

**Comment by br41nslug** [comment]
this has been implemented
